src/Components/layout/header.js

```diff
@@ -23,18 +23,30 @@ const Header = () => {
   return (
     <>
       <nav className=" grid grid-cols-5 p-3">
-        <h1 className=" col-span-2 m-auto flex justify-center">
-          <NavLink
-            to={links[0].path}
-            data-testid="link-item"
-          >
-            <img src={Logo} alt="logo" />
-          </NavLink>
+        <h1 className=" col-span-2 m-auto flex justify-center w-full">
+
+          <div className="flex w-full justify-around">
+            <NavLink
+              to={links[0].path}
+              data-testid="link-item"
+              className=""
+            >
+              <img src={Logo} alt="logo" />
+            </NavLink>
+            <NavLink
+              to={links[0].path}
+              data-testid="link-item"
+              className=""
+            >
+              <h1 className="md:text-2xl hover:text-blue-600 hidden sm:block text-xl font-bold font-italics">Space Travelers Hub</h1>
+
+            </NavLink>
+          </div>
         </h1>
```

Comment:
In the highlighted code, it looks like you might be using one  h1 element to wrap another h1 element.
It is generally not recommended per JavaScript best practices to wrap one h1 element inside another h1 element, as this can create confusion for screen readers and search engines. 
In this case, it might be more appropriate to use a different element, such as a div or a span, to wrap the logo or branding content.